res/test/test-app.html

```diff
@@ -8,8 +8,22 @@
     </script>
     <script>
 
-        function errorHandler(error) {
-            throw fdc3.serializeError(error);
```

Comment:
`serializeError` is intentionally not exported, so we can't straightforwardly reuse code here